examples/resources/netbox_ipam_ip_range/resource.tf

```
@@ -0,0 +1,54 @@
+resource "netbox_ipam_ip_range" "range_test" {
+  start_address = "192.168.56.1/24"
+  end_address = "192.168.56.100/24"
+  description = "Range created by terraform"
+  role_id = data.netbox_ipam_roles.vlan_role_production.id
```

```suggestion
  role_id = data.netbox_ipam_role.vlan_role_production.id
```

Could you please change this on all examples ?

---

Done